tfhub_dev/assets/sayakpaul/mobilenetv2-coco/int8/1.md

```diff
@@ -0,0 +1,26 @@
+# Lite sayakpaul/mobilenetv2-coco/int8/1
```

Comment:
Is the goal of the PR to reorganize models into different directories? If yes, we should keep the first line of each MD file (model identifier) unchanged.

Changing the model identifier (also known as handle) from `sayakpaul/deeplabv3-mobilenetv2-int8/1/default/1` to `sayakpaul/mobilenetv2-coco/int8/1` will break all existing TF Hub users that load the model via url, for example:

`hub.load("https://tfhub.dev/sayakpaul/lite-model/deeplabv3-mobilenetv2-int8/1/default/1")`